.github/workflows/go-channels.yaml

```diff
@@ -0,0 +1,167 @@
+# This workflow resolves the required Go snap channels by considering the
+# following inputs:
+# - (optional) include the Go channel used for building the Snapd snap
+# - (optional) include the Go channel used for building the Snapd snap for FIPS
+# - (optional) the latest stable channel (if version differs from former two)
+# - (optional) list of specific Go channels
+#
+# The output is a single line JSON array that can be used with fromJson(...)
+# to populate YAML arrays.
```

Comment:
This is a nitpick-y personal opinion that you are more than welcome to ignore. This description is just repeating the same things you say below in the workflow name and input descriptions. The repetition could, in case of a code change, cause confusion if only one of those places gets updated.

Reply:
Agreed, the yaml is pretty descriptive and simple enough to follow, so lets not duplicate. Removed.